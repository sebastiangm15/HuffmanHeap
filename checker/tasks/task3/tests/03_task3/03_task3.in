10
15 SATKE
18 SATOZ
26 SATQ7
13 SATB0
7 SATBX
26 SATIG
28 SAT71
8 SATWX
10 SATXU
6 SAT7V
6
SATBX
SATXU
SATIG
SAT7V
SATKE
SATQ7
