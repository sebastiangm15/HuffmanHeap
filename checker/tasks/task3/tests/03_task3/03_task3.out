10101100111010100010111
